internal/pkg/template/templates/workloads/services/worker/manifest.yml

```diff
@@ -28,7 +28,7 @@ image:
 cpu: {{.CPU}}       # Number of CPU units for the task.
 memory: {{.Memory}}    # Amount of memory in MiB used by the task.
 {{- if .Platform}}
-platform: {{.Platform}}
+platform: {{.Platform.PlatformString}}
```

Comment:
https://github.com/aws/copilot-cli/blob/0d09b8ab883d3800f870e17194cf670dfc860595/internal/pkg/cli/job_init.go#L202

Let me know if I'm missing something - seems like `.Platform` ever `nil`, is that correct?

Reply:
Fixed.